resources/public/api/conf/1.0/errors.raml

```diff
@@ -24,6 +24,16 @@ traits:
               description: Employments could not be found for the supplied NINO.
               value:
                 code: MATCHING_RESOURCE_NOT_FOUND
+  notFoundExpenses:
```

Comment:
Raised a question in the RAD channel about the wording of this - wait to see what people say about it. Otherwise it looks good, I'll just need to run the ATs